<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadLineItem</class>
 <widget class="QWidget" name="UploadLineItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="imagePreview">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="uploadTimestamp">
       <property name="text">
        <string notr="true">timestamp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyUrl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Copy URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openBrowser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open In Browser</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteImage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../data/graphics.qrc">
         <normaloff>:/img/material/black/delete.svg</normaloff>:/img/material/black/delete.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data/graphics.qrc"/>
 </resources>
 <connections/>
</ui>
